README.md

```diff
@@ -89,21 +89,21 @@ Unlike many authorization systems where a service controls access to resources i
 
 ## 2.1 Resource
 
-A resource is some data or process that has an address. It can be anything from a row in a database, a user account, storage quota, email address, and so on.
+A resource is some data or process that has an address. It can be anything from a row in a database, a user account, storage quota, email address, etc.
 
 ## 2.2 Action
 
-An action is performed against some resource. Each action MAY have its own semantics. They MAY be unary, support a hierarchy, be monotone, partial orders, and so on. Actions MAY be general and applicable to many kinds of resource, or tied to a specific one.
+An action MUST be performed against some resource. Each action MAY have its own semantics. For example, actions MAY be unary, support a hierarchy, be monotone, form partial orders, etc. In addition, actions MAY be general and applicable to many kinds of resources or tied to a specific one.
 
-For instance, `wnfs/APPEND` is an action for WebNative filesystem paths. The action `wnfs/OVERWRITE` also implies the capacity to append. Email has no such tiered relationship. One can `email/SEND`, but there is no concept of a ”super send”.
+For instance, `wnfs/APPEND` is an action for WebNative filesystem paths. The action `wnfs/OVERWRITE` also implies the capacity to append. An email has no such tiered relationship. One can `email/SEND`, but there is no concept of a" super send."
```

Comment:
> but there is no concept of ~a" super send."~ a "super send."

Add a space after "a" and remove one from before "super".

Reply:
Woah weird. I wonder if there's a mismatched quote somewhere? Looks like that happened as part of the conversion to consistent quotes.

Good catch!